submissions/kolawoleOgunfowokan/PHPTask/php/create.php

```diff
@@ -0,0 +1,61 @@
+<?php
+if (isset($_POST['submit'])) {
+    require "config.php";
+    require "helper.php";
+
+    try {
+        $connection = new PDO($dsn, $username, $password, $options);
+
+        $newProduct = array(
+            "itemname" => $_POST['itemname'],
+            "quantity" => $_POST['quantity'],
+            "brand_id" => $_POST['brand'] + 1
+        );
+
+        /**
+         * This is equivalent to
+         * INSERT INTO products (itemname, quantity) values (:itemname, :quantity)
+         */
+        $sql = sprintf(
+            "INSERT INTO %s (%s) values (%s)",
+            "products",
+            implode(", ", array_keys($newProduct)),
+            ":" . implode(", :", array_keys($newProduct))
+        );
+
+        $statement = $connection->prepare($sql);
+        $statement->execute($newProduct);
+    } catch (PDOException $error) {
+        echo $sql . "<br>" . $error->getMessage();
+    }
+}
+?>
+
+<?php if (isset($_POST['submit']) && $statement) { ?>
+    > <?php echo $_POST['itemname']; ?> successfully added.
+<?php } ?>
+
+
+<h2>Add A Product </h2>
+
+<form method="post">
+    <div class="form-container">
+        <label for="itemname">Name of Product</label>
+        <input type="text" name="itemname" id="itemname">
+        <?php session_start(); ?>
+
+        <select name="brand">
+            <?php for ($i = 0; $i < count($_SESSION['brands']); $i++) { ?>
```

Comment:
Hey @kolawolejohn , I like the idea of using sessions to store the brands but from what I see, this value gets populated from `fetch.php` which is not loaded on the `create.php` page. So anytime this page gets loaded, the brands are empty. 

Maybe you should include the `fetch.php` page on this file as well.

Reply:
Fixed and quite good, thanks for pointing it out.